tests/laser/state/calldata_test.py

```diff
@@ -0,0 +1,80 @@
+import pytest
```

Comment:
One more interesting situation might be the following:

calldatasize == 50
calldata[55] == 0

Then solver.check should return unsat

Reply:
I'll add that.